ios/Plugin/InAppReview.swift

```diff
@@ -4,7 +4,22 @@ import StoreKit
 
 @objc public class InAppReview: NSObject {
     @objc func requestReview(_ call: CAPPluginCall) {
-        SKStoreReviewController.requestReview()
+        DispatchQueue.main.async {
+            if #available(iOS 17.0, *) {
+                if let windowScene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
+                    AppStore.requestReview(in: windowScene)
+                }
+            } else if #available(iOS 14.0, *) {
+                if let windowScene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
+                    SKStoreReviewController.requestReview(in: windowScene)
+                }
+            } else {
+                SKStoreReviewController.requestReview()
+            }
```

Comment:
```suggestion
            } else {
                if let windowScene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
                    SKStoreReviewController.requestReview(in: windowScene)
                }
            }
```
The plugin only supports iOS 14+ now, so the last else can be removed.

And you can also use a single `if let windowScene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene` before the if instead of duplicating it for each case since the code is the same.

Reply:
Yes i agree, thank you

Reply:
Can you address the second part of the comment?